File: buildSrc/src/main/java/com/doctor/aistring/KeyGenerator.java
package com.doctor.aistring;

public class KeyGenerator {
    public static String generateKey() {
        byte[] key = new byte[32]; // 256位密钥
        new java.security.SecureRandom().nextBytes(key);
        StringBuilder hexString = new StringBuilder();
        for (byte b : key) {
            hexString.append(String.format("%02x", b));
        }
        return hexString.toString();
    }
}

File: buildSrc/src/main/kotlin/com/doctor/aistring/KeyGenerator.kt
package com.doctor.aistring

import java.io.File
import java.security.SecureRandom
import java.util.Base64

object KeyGenerator {
    fun generateKey(): String {
        val key = ByteArray(32) // 256位密钥
        java.security.SecureRandom().nextBytes(key)
        return key.joinToString("") { "%02x".format(it) } // 转换为十六进制字符串
    }
}

